doc/pagerank.adoc

```diff
@@ -14,12 +16,16 @@ We assume page A has pages T1...Tn which point to it (i.e., are citations). The
 
 PR(A) = (1-d) + d (PR(T1)/C(T1) + ... + PR(Tn)/C(Tn))
 
-Note that the PageRanks form a probability distribution over web pages, so the sum of all web pages' PageRanks will be one.
```

Comment:
do you rather want to move these more graph-theory notes that you removed to a dedicated section